4-Server/server.js

```diff
@@ -9,16 +9,35 @@ app.use(cookieParser());
 
 app.use(express.json());
 
-app.use(express.static(__dirname + '../3-JSS/index.html'));
 
 const redis = require('redis');
 const client = redis.createClient(process.env.REDIS_URL);
 
 let currentUserId;
 
 app.get('/', (req, res) => {
-    res.sendFile(path.resolve(__dirname, '../3-JSS/index.html'));
-  });
+    res.sendFile(path.resolve(__dirname, '../3-JSS/src/index.html'));
```

Comment:
this is not scalable.. for each new file u will need to add an endpoint here.
Instead you should mount the dist folder. You should make sure to copy the index.html and other assets (aka the assets folder) to the dist in build time.